ledger/participant-integration-api/src/main/scala/platform/apiserver/services/transaction/ApiTransactionService.scala

```diff
@@ -264,4 +263,15 @@ private[apiserver] final class ApiTransactionService private (
       logging.workflowId(workflowId),
       logging.offset(offset),
     )
+
+  override def getLatestPrunedOffsets: Future[GetLatestPrunedOffsetsResponse] =
+    transactionsService.latestPrunedOffsets().map {
+      case (prunedUpToInclusive, divulgencePrunedUpTo) =>
+        GetLatestPrunedOffsetsResponse(
+          participantPrunedUpToInclusive =
+            Some(LedgerOffset(LedgerOffset.Value.Absolute(divulgencePrunedUpTo.value))),
```

Comment:
I think the offsets are mixed up

Reply:
fortunately just in the naming - due to a double-renaming error. They end up fine in the API. Will adapt the names, thanks!